crates/database/src/auction_participants.rs

```diff
@@ -41,9 +42,34 @@ pub async fn fetch(
     sqlx::query_as(QUERY).bind(auction_id).fetch_all(ex).await
 }
 
+#[derive(sqlx::FromRow)]
+pub struct ParticipantAddress(pub Address);
+
+pub fn latest_auction_participants(
+    ex: &mut PgConnection,
+) -> BoxStream<'_, Result<ParticipantAddress, sqlx::Error>> {
+    const QUERY: &str = r#"
+SELECT participant
+FROM auction_participants
```

Comment:
This table contains the solvers that submitted a solution for an auction which is unrelated to the problem at hand.